kernel-updates/get-kernel-updates.sh

```diff
@@ -45,24 +92,50 @@ source "./yaml-parser.sh"
 create_variables endpoints.yml yaml
 rm -f ./endpoints.yml
 
-curl -L -o latest-kernel "https://github.com/netbootxyz${yamlendpoints_ubuntu_23_04_KDE_squash_path}vmlinuz"
+curl -L -o vmlinuz "https://github.com/netbootxyz${yamlendpoints_ubuntu_23_04_KDE_squash_path}vmlinuz"
 # Determine the kernel version of the just downloaded kernel, e.g. "5.4.0-42-generic"
-kernelVersion=$(file -b latest-kernel | grep -o 'version [^ ]*' | cut -d ' ' -f 2)
+kernelVersion=$(file -b vmlinuz | grep -o 'version [^ ]*' | cut -d ' ' -f 2)
 echo "Newest kernel has version $kernelVersion"
 
+# Maps the filename to the target folder, where it will be copied to
+declare -A file_map
+file_map["vmlinuz"]="kernels/$kernelVersion"
+file_map["initrd"]="kernels/$kernelVersion"
+file_map["latest-kernel-version.json"]="kernels"
 kernelIsNew=$(isKernelNewer "$kernelVersion" "$netbootIP")
```

Comment:
could be directly move to the if condition as in `if isKernelNewerThanOnBlobStorage "$kernelVersion";` ?